app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_status_validation_mixin.py

```diff
@@ -56,3 +58,308 @@ def test_update_experiment_with_invalid_publish_status_error(self):
+        [
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.IDLE,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.DIRTY,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.REVIEW,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.APPROVED,
+                False,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.WAITING,
+                False,
+            ],
+        ]
+    )
+    def test_update_publish_status_errors_for_create(self, lifecycle, status, valid):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            lifecycle,
+        )
+        serializer = NimbusExperimentSerializer(
+            experiment,
+            data={
+                "publish_status": status,
+                "changelog_message": "test changelog message",
+            },
+            context={"user": self.user},
+        )
+        self.assertEquals(serializer.is_valid(), valid)
+
+    @parameterized.expand(
+        [
+            [
+                NimbusExperimentFactory.Lifecycles.PREVIEW,
+                NimbusExperiment.PublishStatus.IDLE,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.PREVIEW,
+                NimbusExperiment.PublishStatus.DIRTY,
+                True,
+            ],  # should this be False?
+            [
+                NimbusExperimentFactory.Lifecycles.PREVIEW,
+                NimbusExperiment.PublishStatus.REVIEW,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.PREVIEW,
+                NimbusExperiment.PublishStatus.APPROVED,
+                False,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.PREVIEW,
+                NimbusExperiment.PublishStatus.WAITING,
+                False,
+            ],
+        ]
+    )
+    def test_update_publish_status_errors_for_preview(self, lifecycle, status, valid):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            lifecycle,
+        )
+        serializer = NimbusExperimentSerializer(
+            experiment,
+            data={
+                "publish_status": status,
+                "changelog_message": "test changelog message",
+            },
+            context={"user": self.user},
+        )
+        self.assertEquals(serializer.is_valid(), valid)
+
+    @parameterized.expand(
+        [
+            [
+                NimbusExperimentFactory.Lifecycles.LIVE_IDLE,
```

Comment:
Don't need LIVE_IDLE here

Reply:
Oh there's more, yeah if it's the same in all of them then it doesn't need to be a parameter.